### PR TITLE
Add InitialQuery event

### PR DESCRIPTION
**Pull request by Unalian**
### Changelog category:
- Improvement

### Changelog entry:
Add improvement to count InitialQuery

### Documentation entry for user-facing changes:

- Motivation: Add InitialQuery event. Similar to counting Queries, but only counting initial queries. Event Contains information about the number of events that have occurred in the system. For example, in the table, you can find how many initial queries were processed since the ClickHouse server started. 

- Example use: 
```SQL
SELECT * FROM system.events where event = 'InitialQuery'
```

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: src/Databases/DatabaseReplicatedWorker.cpp [hunk 1]
```diff
@@ -264,7 +267,7 @@ String DatabaseReplicatedDDLWorker::tryEnqueueAndExecuteEntry(DDLLogEntry & entr
     chassert(!task->entry.query.empty());
     assert(!zookeeper->exists(task->getFinishedNodePath()));
     task->is_initial_query = true;
-
+    ProfileEvents::increment(ProfileEvents::InitialQuery);
```
Comment (alexey-milovidov) [inline src/Databases/DatabaseReplicatedWorker.cpp:270]: It's not the only place where it is set.
Better, check `executeQuery.cpp`

**Comment by robot-ch-test-poll** [comment]
*This is an automated comment for commit a6f2eaf5a6ba2a26943d0c1c53c7cf7460a7471d with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/58195/a6f2eaf5a6ba2a26943d0c1c53c7cf7460a7471d/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>❌ failure</td></tr>
<tbody>
</table>

**Comment by alexey-milovidov** [comment]
@Unalian, the test script has no executable permission.

**Comment by Unalian** [comment]
> @Unalian, the test script has no executable permission.

Sorry for that. I have fixed it.

**Comment by Unalian** [comment]
@alexey-milovidov Hi, could you please tell me what happened to the PR? It seems like there is something wrong because it shows "Some checks were not successful" but I can not find the problem description.

**Review by alexey-milovidov — APPROVED** [review]
The code LGTM.

**Pull request merged**